systemvm/patches/debian/config/opt/cloud/bin/vr_cfg.sh

```diff
@@ -93,4 +93,16 @@ done < $cfg
 #remove the configuration file, log file should have all the records as well
 rm -f $cfg
 
+# Flush kernel conntrack table
+log_it "VR config: Flushing conntrack table"
+ps cax | grep conntrackd > /dev/null
+if [ $? -eq 0 ]; then
```

Comment:
instead of ps <args> | grep, why not use status result and start, like: service conntrackd status || service conntrackd start (or other commands) etc.